### PR TITLE
Fix build script

### PR DESCRIPTION
**Pull request by Pessimistress**
https://github.com/uber-web/math.gl/issues/147

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,8 +13,7 @@
   ],
   "scripts": {
     "bootstrap": "PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true yarn && ocular-bootstrap",
-    "clean": "ocular-clean",
```
Comment (ibgreen) [inline package.json:16]: I guess I need to rely on `git clean` now to get rid of built stuff. That is probably fine, pretty rare to need clean.

Should we consider updating ocular build to also call clean?

Comment (Pessimistress) [reply package.json:16]: I recall you feel strongly about separating clean and build.

Comment (ibgreen) [reply package.json:16]: Ha ha, yes, but for some reason, I don't seem to feel as strongly about it anymore. My current thoughts on the topic:

- I see a value in having a separate `yarn clean`, to get back to a fresh state.
- but I also see a value in having less scripts. 

So I am happy to drop it. There is always `git clean`. We could also support `yarn build clean` as an option in ocular dev tools, less scripts in each package.json.

The one "exception" is in loaders.gl which has a very long and slow build process, where there is a need to build workers to get tests running at all, it has been valuable to be able to rebuild individual modules without cleaning the entire tree. 

But it is a corner case, and we can just make sure to not call clean when specifying a module to build.
